### PR TITLE
Add processor to lazy load the videos

### PR DESCRIPTION
**Pull request by SantosGuillamot**
In order to improve the performance, we want to lazy load the videos of the web. There's a video in the homepage named `LiveContentUpdate` where we can check this.

### CODE REVIEW

**Comment by SantosGuillamot** [comment]
@DAreRodz It seems to be working but I had to do the commit with `--no-verify` because I was getting this error:
```
Media elements such as <audio> and <video> must have a <track> for captions  jsx-a11y/media-has-caption
```

But I've check and it seems that WordPress isn't adding `<track>`. Do you think this is something we should solve?

**Comment by frontibotito** [comment]
Deploy preview for _website_ ready!

Built with commit 97a36594dde903b9bd682381a73212f6f4f0f464

https://frontity-org-46hkgsli8.now.sh

File: packages/frontity-org-theme/src/components/lazy-video.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import useInView from "@frontity/hooks/use-in-view";
```
Comment (DAreRodz) [inline packages/frontity-org-theme/src/components/lazy-video.tsx:1]: I would disable the rule `jsx-a11y/media-has-caption` for this file, using the following line at the top:
```ts
/* eslint-disable jsx-a11y/media-has-caption */
```

Comment (SantosGuillamot) [reply packages/frontity-org-theme/src/components/lazy-video.tsx:3]: Done, thanks David :slightly_smiling_face:

**Pull request merged**